orttraining/orttraining/training_ops/cuda/tensor/slice_grad.cc

```diff
@@ -50,7 +50,7 @@ void SliceGrad::FillInputVectors(OpKernelContext* ctx, std::vector<int64_t>& inp
 Status SliceGrad::CallSliceImp(size_t element_size, size_t dimension_count, const TArray<int64_t>& starts_buffer,
                                const TArray<int64_t>& steps_buffer, const TArray<int64_t>& input_strides,
                                const TArray<fast_divmod>& output_strides, OpKernelContext* ctx,
-                               TensorShape output_shape) const {
+                               TensorShape& output_shape) const {
   Tensor* gradient_out_tensor = GetOutputGradientTensor(ctx);
```

Comment:
const TensorShape& given it is only used to get the rank

Reply:
done